### PR TITLE
Mark the dependencies listed in project.json in kpm list

### PR DESCRIPTION
**Pull request by troydai**
Address https://github.com/aspnet/XRE/issues/1098

### CODE REVIEW

**Comment by troydai** [comment]
@Praburaj

File: src/Microsoft.Framework.PackageManager/List/LibraryDependencyFlatRenderer.cs [hunk 1]
```diff
@@ -71,16 +73,17 @@ private void RenderLibraries(IEnumerable<Library> libraries, IDictionary<Library
 
             foreach (var lib in libraries)
             {
+                var libDisplay = (_listedProjects.Contains(lib.Name) ? "*" : " ") + lib.ToString();
```
Comment (Praburaj) [inline src/Microsoft.Framework.PackageManager/List/LibraryDependencyFlatRenderer.cs]: `string.empty` instead of `" "`?

Comment (Praburaj) [inline src/Microsoft.Framework.PackageManager/List/LibraryDependencyFlatRenderer.cs]: Alternatively avoid string concat for non-listed projects. 
```
var libDisplay = _listedProjects.Contains(lib.Name) ? 
                           "*" + lib.ToString() : 
                           lib.ToString();
```

**Comment by Praburaj** [comment]
:shipit: after these comments addressed.

**Comment by davidfowl** [comment]
Uh, this PR looks ruined.

File: src/Microsoft.Framework.PackageManager/List/LibraryDependencyFlatRenderer.cs (hunk 1 above)
Comment (troydai) [inline src/Microsoft.Framework.PackageManager/List/LibraryDependencyFlatRenderer.cs]: I was thinking using one addition space to align the output.

**Comment by troydai** [comment]
@davidfowl looks better now.

**Comment by troydai** [comment]
Ping @davidfowl

**Pull request merged**